ceph-salt-formula/salt/ceph-salt/files/registries.conf.j2

```diff
@@ -0,0 +1,18 @@
+{% set registries = pillar['ceph-salt'].get('container', {}).get('registries', []) -%}
+# System Registry Configuration File (VERSION 2)
+
+# An array of host[:port] registries to try when pulling an unqualified image, in order
+unqualified-search-registries = ["docker.io"]
+
+{% for registry in registries -%}
+[[registry]]
+{%- set reg = registry | load_json %}
+prefix = "{{ reg.prefix }}"
```

Comment:
`prefix` is optional (see `man containers-registries.conf`). So it might not be defined.

Reply:
Fixed.